### PR TITLE
working press page and youtube link

### PR DESCRIPTION
**Pull request by thenomemac**
press page and youtube link on site

### CODE REVIEW

**Comment by thenomemac** [comment]
also added beta to site header and a "features under construction warning on home page"

**Comment by MattGraz** [comment]
Not sure if I am getting the following because I'm running local or something but it looks like a couple images may not be coming through. 

screen shot 2017-04-18 at 12 05 18 am

Also, "Download Data" appears twice in the selection.

Also, lets plan on getting an updated video uploaded before the presentation/launch of the webpage #59

File: fishr/templates/index.html [hunk 1]
```diff
@@ -57,6 +66,32 @@
...
+    </div>
+  </div>
+
+
+  <div class="col-xs-6">
+    <div class="thumbnail text-center">
+      <a href="{{ url_for('about') }}">
+        <img class="img-responsive icon" src="{{ url_for('static', filename='images/press_icon.png') }}" alt="About Us">
+        <div class="caption">
+            <p>Download Data</p>
```
Comment (MattGraz) [inline fishr/templates/index.html]: I this is meant to be "About us"

**Comment by thenomemac** [comment]
Ya was supposed to be about. Also I'll update the vid before finals. I'll
look into why the images aren't coming through. B

**Comment by thenomemac** [comment]
okay fixed everything up. needs another review

**Pull request merged**